crates/matrix-sdk-base/src/client.rs

```diff
@@ -169,6 +175,7 @@ impl BaseClient {
     pub async fn clone_with_in_memory_state_store(
         &self,
         cross_process_store_locks_holder_name: &str,
+        handle_verification_events: bool,
```

Comment:
I don't think that this necessarily needed to be an argument for this function, after all, we're only using it with the `false` parameter.

I guess some other unlikely user of this method might want to set it to `true`.

Reply:
So should I get rid of the parameter? I left it both because handling verifications doesn't have much to do with cloning the client so it's a bit weird that we assume it must be false, and also in case some user of the SDK had any use case in mind that needed the parameter to be true.

Reply:
Let's leave it in.

It's very likely that this function is only used to create a notification client object, then we could rename the method to `notification_client()`.

I'm not completely certain that this is true, so let's leave it as it is.